doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -43,11 +54,7 @@ Traceability to Reference Model.
 |----|---------------|---------------------|---------------|
 | `req.gen.ost.01` | Open source | The Architecture **must** use OpenStack APIs.| [RA-1 5.3](./chapter05.md#53-consolidated-set-of-apis) |
 | `req.gen.ost.02` | Open source | The Architecture **must** support dynamic request and configuration of virtual resources (compute, network, storage) through OpenStack APIs. | [RA-1 5.3](./chapter05.md#53-consolidated-set-of-apis) |
-| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. | OpenStack consists of both stateless and stateful services where the stateful services utilize a database. For latter see "[Configuring the stateful services](https://docs.openstack.org/ha-guide/control-plane-stateful.html)"|
```

Comment:
Sequence numbers needs fixings

Reply:
@rabi-abdel the numbering reflects the sub-category; this practice is inherited for RM